integrations/vello_svg/src/geom.rs

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// copied from https://github.com/RazrFalcon/resvg/blob/4d27b8c3be3ecfff3256c9be1b8362eb22533659/crates/resvg/src/geom.rs

```

Comment:
I don't really know the copyright/license problem here. Is it ok to use this code? And ci looks not happy with this.

Reply:
Yes it is ok. The MPL 2.0 license is compatible enough with Apache 2.0 OR MIT, and we already import `usvg` which is licensed under MPL 2.0 so this has no effect on the license mixture of `vello_svg`.

We can replace the copyright notice header of this file with a different format, which I showed in a sibling comment.

You will also need to edit `.github/copyright.sh` and look for the first long regex line that starts with `output=`. Replace that line with the following:

```sh
output=$(rg "^// Copyright (19|20)[\d]{2} (.+ and )?the Vello Authors( and .+)?$\n^// SPDX-License-Identifier: Apache-2\.0 OR MIT$\n\n" --files-without-match --multiline -g "*.rs" -g "!{shader,src/cpu_shader}" -g "!integrations/vello_svg/src/geom.rs" .)
```